crates/wasmcloud-host/src/host.rs

```diff
@@ -42,6 +43,14 @@ impl HostBuilder {
             rpc_timeout: Duration::from_secs(2),
             rpc_client: None,
             cplane_client: None,
+            allow_live_update: false,
+        }
+    }
+
+    pub fn enable_live_updates(self) -> HostBuilder {
+        HostBuilder {
+            allow_live_update: true,
+            ..self
```

Comment:
Something like this was what I was referring to in my previous comment about not allowing updates with increased claims.